grails-app/services/org/pih/warehouse/inventory/ProductInventoryTransactionService.groovy

```diff
@@ -117,6 +117,7 @@ abstract class ProductInventoryTransactionService<T> {
         for (AvailableItem availableItem : availableItems) {
             TransactionEntry transactionEntry = new TransactionEntry(
                     quantity: availableItem.quantityOnHand,
+                    product: availableItem.inventoryItem.product,
```

Comment:
I have no idea how this was working before. This was always missing from the start... :shrug:

Reply:
I think it was working because other parts rely on inventory item but I am not 100% sure atm.